api/commands/build.ts

```diff
@@ -61,8 +61,8 @@ export class Builder {
 
     // Determine which nodes should be included.
     const includedNodeNames =
```

Comment:
`includedActionNames` now?

(actually there are a lot of usages of the term 'node' both before and after this code, can you replace them with 'action' to be consistent please?)

Reply:
I have only replaced those place which is related to actions. There are some places where nodes are referred to the execution graph nodes.